app/lib/features/news/providers/news_post_editor_providers.dart

```diff
@@ -40,6 +40,12 @@ class NewsStateNotifier extends StateNotifier<NewsPostState> {
     );
   }
 
+  void setSpaceId(String? spaceId) {
+    state = state.copyWith(
```

Comment:
Can we add unit test for this case so to ensure it saves up correct spaceId when switching spaces?

Reply:
this one-liner you want a test case for?